API/Services/DirectoryService.cs

```diff
@@ -259,7 +260,12 @@ public bool IsDriveMounted(string path)
            return FileSystem.DirectoryInfo.FromDirectoryName(FileSystem.Path.GetPathRoot(path) ?? string.Empty).Exists;
        }
 
-       public string[] GetFilesWithExtension(string path, string searchPatternExpression = "")
+        public bool IsDirectoryEmpty(string path)
```

Comment:
Add a unit test for this please

Reply:
Added in https://github.com/Kareadita/Kavita/pull/871/commits/a0fa4fb2800b5881b97000c4b73a59de3de68d7d

Resolving request.